examples/requirements.txt

```diff
@@ -1,8 +1,15 @@
-itk-elastix>=0.13.0
+itk-elastix>=0.19.1
 itkwidgets>=0.32.0
 jupyterlab>=2.2.0
 imageio
 ipywidgets>=7.5.1
 ipympl>=0.5.7
-voila>=0.3.4
-pyinstaller
+numpy
+torch>=2.0
+monai>=1.2.0
+matplotlib==3.3.1
+PyQt5==5.15.0
```

Comment:
Perhaps changes from #230 should be considered here.

Reply:
Do you know where we are using PyQt? Not sure why this is a dependency?

Reply:
I simply copy/pasted this from `binder/requirements.txt`, maybe not all of these are required for simple notebook testing.

Reply:
Let's try removing it, as per #185.